project-docs/ADMINS.md

```diff
@@ -0,0 +1,27 @@
+# Admins
+
+This document lists the Admins of the Project. Admins may be added once approved by the existing Admins as described in the [Governance document](GOVERNANCE.md). By adding your name to this list you are agreeing to abide by the Project governance documents and to abide by all of the Organization's polices, including the [code of conduct](CODE_OF_CONDUCT.md).
+
+## Vega and Vega-Lite Admins
+
+This includes projects in the [vega](https://github.com/vega/) org.
+
+| **NAME** | **Handle** | **Affiliated Organization** |
+| --- | --- | --- |
+| Arvind Satyanarayan | @arvind | - |
+| Dominik Moritz | @domoritz | - |
+| Jeffrey Heer | @jheer | - |
+| Kanit Wongsuphasawat | @kanitw | - |
+| Lukas Hermann | @lsh | - |
+
+## Vega Altair Admins
```

Comment:
I think we've been using a hyphen in the name, so Vega-Altair.

Reply:
Done